Ontology/Asset/Equipment/HVACEquipment/AirHandlingUnit.json

```diff
@@ -2,91 +2,29 @@
   "@id": "dtmi:digitaltwins:rec_3_3:asset:AirHandlingUnit;1",
   "@type": "Interface",
   "contents": [
-    {
-      "@type": "Property",
-      "displayName": {
-        "en": "primary heating method"
-      },
-      "name": "primaryHeatingMethod",
-      "dtmi:dtdl:property:schema;2": {
-        "@type": "Enum",
-        "dtmi:dtdl:property:enumValues;2": [
-          {
-            "enumValue": "Electric",
-            "name": "Electric"
-          },
-          {
-            "enumValue": "Gas",
-            "name": "Gas"
-          },
-          {
-            "enumValue": "HotWater",
-            "name": "HotWater"
-          },
-          {
-            "enumValue": "None",
-            "name": "None"
-          }
-        ],
-        "valueSchema": "string"
-      },
-      "writable": true
-    },
     {
       "@type": "Property",
       "displayName": {
         "en": "primary cooling method"
       },
       "name": "primaryCoolingMethod",
-      "dtmi:dtdl:property:schema;2": {
-        "@type": "Enum",
-        "dtmi:dtdl:property:enumValues;2": [
-          {
-            "enumValue": "None",
-            "name": "None"
-          },
-          {
-            "enumValue": "DX",
-            "name": "DX"
-          },
-          {
-            "enumValue": "ChilledWater",
-            "name": "ChilledWater"
-          }
-        ],
-        "valueSchema": "string"
-      },
-      "writable": true
+      "schema": "dtmi:digitaltwins:rec_3_3:asset:HVACCoolingMethod;1"
     },
     {
       "@type": "Property",
       "displayName": {
         "en": "pre-heating method"
       },
       "name": "preHeatingMethod",
-      "dtmi:dtdl:property:schema;2": {
-        "@type": "Enum",
-        "dtmi:dtdl:property:enumValues;2": [
-          {
-            "enumValue": "None",
-            "name": "None"
-          },
-          {
-            "enumValue": "HotWater",
-            "name": "HotWater"
-          },
-          {
-            "enumValue": "Gas",
-            "name": "Gas"
-          },
-          {
-            "enumValue": "Electric",
-            "name": "Electric"
-          }
-        ],
-        "valueSchema": "string"
+      "schema": "dtmi:digitaltwins:rec_3_3:asset:HVACHeatingMethod;1"
```

Comment:
A Property cannot have a schema that is an interface. This element could be changed to be a Component or the schema defined inline or in a reusable schema in the Interface's schemas element.